topics/scikit-learn/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David @cournape Cournapeau, The scikit-learn developers
```

Comment:
We just use names (not handles), could you edit to the following?

> David Cournapeau

Reply:
Sorry, no. Could you do that for me?

Reply:
@KOLANICH Ping on this.

Reply:
Fixed.